.ci/Jenkinsfile

```diff
@@ -24,6 +24,7 @@ pipeline {
     issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
   }
   parameters {
+    choice(name: 'runTestsSuite', choices: ['all', 'helm', 'metricbeat'], description: 'Choose which test suite to run (default: all)')
```

Comment:
Nit: It seems impossible to run helm+metricbeat but not all. Is that intentional?

Reply:
I don't have any opinion about what's right here. :) I was just raising the point in case it was something you wanted to handle. Thanks!

Reply:
Sure! At this moment I could not say we are not going to need it, although given the current  parallel implementation of the CI in separate workers, I would not see potential collision issues while running different combinations of them.